go.mod

```diff
@@ -3,7 +3,8 @@ module github.com/mattermost/mattermost-plugin-msteams-meetings
 go 1.12
 
 require (
-	github.com/mattermost/mattermost-server/v5 v5.27.0
+	github.com/mattermost/mattermost-plugin-api v0.0.12-0.20200908141642-fbaca40730c4
```

Comment:
Nit pick: I think you can use the latest release `v0.0.12` directly here

Reply:
I'll make that change, thank you for pointing that out

Reply:
Could we do the same with https://github.com/mattermost/mattermost-plugin-msteams-meetings/pull/18/files#diff-37aff102a57d3d7b797f152915a6dc16R7